winrt-related-src/toolkits/winui3/index.md

```diff
@@ -148,13 +148,15 @@ The following list shows the UI controls and features that are not included (or
 * [MediaPlayerElement](https://docs.microsoft.com/uwp/api/windows.ui.xaml.controls.mediaplayerelement), [MediaElement](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.MediaElement) and related APIs
 * [RevealBrush](https://docs.microsoft.com/uwp/api/windows.ui.xaml.media.revealbrush)
 * [SwapChainPanel](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.SwapChainPanel) and [SwapChainBackgroundPanel](https://docs.microsoft.com/uwp/api/windows.ui.xaml.controls.swapchainbackgroundpanel)
-* [System.ComponentModel.INotifyPropertyChanged](https://docs.microsoft.com/dotnet/api/system.componentmodel.inotifypropertychanged?view=dotnet-uwp-10.0) and [System.Collections.Specialized.INotifyCollectionChanged](https://docs.microsoft.com/dotnet/api/system.collections.specialized.inotifycollectionchanged?view=dotnet-uwp-10.0) for C# apps. Instead, use the ```Microsoft.UI.Xaml.Interop``` versions of these interfaces
 * [ThemeShadow](https://docs.microsoft.com/uwp/api/windows.ui.xaml.media.themeshadow)
 * [WebView](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.WebView)
 * [Window.SetTitleBar](https://docs.microsoft.com/uwp/api/windows.ui.xaml.window.settitlebar) and custom window title bars
 * [Xaml Islands](https://docs.microsoft.com/windows/apps/desktop/modernize/xaml-islands) functionality including [DesktopWindowXamlSource](https://docs.microsoft.com/uwp/api/windows.ui.xaml.hosting.desktopwindowxamlsource)
 * [XamlLight](https://docs.microsoft.com/uwp/api/windows.ui.xaml.media.xamllight)
 
+### INotifyPropertyChanged and INotifyCollectionChanged
+The .NET interfaces [System.ComponentModel.INotifyPropertyChanged](https://docs.microsoft.com/dotnet/api/system.componentmodel.inotifypropertychanged?view=dotnet-uwp-10.0) and [System.Collections.Specialized.INotifyCollectionChanged](https://docs.microsoft.com/dotnet/api/system.collections.specialized.inotifycollectionchanged?view=dotnet-uwp-10.0) are projected into the WinRT Windows.UI.Xaml namespace as [Windows.UI.Xaml.Data.INotifyPropertyChanged](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.data.inotifypropertychanged) and [Windows.UI.Xaml.Interop.INotifyCollectionChanged](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.interop.inotifycollectionchanged), respectively. However, WinUI 3.0 depends on interfaces under the Microsoft.UI.Xaml namespace, so types that implement these .NET interfaces do not function well with WinUI 3.0. To work around this issue, your type should implement the corresponding Microsoft.UI.Xaml interfaces.
+
```

Comment:
This is good. Perhaps also mention that using ObservableCollection and making changes to it will cause the wrong thread error.

Reply:
And link to XCG's TestObservableCollection.

Reply:
@MikeHillberg , @ranjeshj: this branch has been merged and deleted. Can you add your suggestions to a new branch off the current master? Thanks!